library/headers.inc.php

```diff
@@ -37,4 +37,5 @@ function include_js_library($path)
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
```

Comment:
@nileshprasad137  you are the lead dog on this. Come up with a good way of doing it. A word of wisdom (I think) "Don't paint yourself in to a corner" Think thru it and don't settle for just getting it done.